server/conf/routes

```diff
@@ -22,3 +22,5 @@ POST    /register                   controllers.UserController.postRegister
 #Login
 GET     /login                      controllers.UserController.getLogin
 POST    /login                      controllers.UserController.postLogin
+GET     /githublogin                controllers.GitHubUser.getLogin
+GET     /githubback                 controllers.GitHubUser.githubCallback(code: String)
```

Comment:
Wow GitHub redirects to `GET /githubback?code=11111111&state=some_state`? It's unsecure, but give us ability to test app on localhost?